src/m365/aad/commands/m365group/m365group-remove.ts

```diff
@@ -69,29 +74,18 @@ class AadM365GroupRemoveCommand extends GraphCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const removeGroup = async (): Promise<void> => {
-      if (this.verbose) {
-        await logger.logToStderr(`Removing Microsoft 365 Group: ${args.options.id}...`);
-      }
+    if (this.verbose) {
+      await logger.logToStderr(`Removing Microsoft 365 Group: ${args.options.id}...`);
+    }
 
+    const removeGroup = async (): Promise<void> => {
       try {
-        const requestOptions: CliRequestOptions = {
-          url: `${this.resource}/v1.0/groups/${args.options.id}`,
-          headers: {
-            'accept': 'application/json;odata.metadata=none'
-          }
-        };
-
-        await request.delete(requestOptions);
+        const siteUrl = await this.getM365GroupSiteURL(logger, args.options.id);
+        await this.deleteM365GroupSite(logger, siteUrl);
```

Comment:
Am I understanding it correctly, that deleting the group site will also delete the group? If so, let's clarify this in a comment